doc/Learn/Markup/Overview.md

```diff
@@ -16,6 +16,17 @@ You will quickly discover why C# Markup is a developer favorite with:
 - Refactoring support
 - Custom Controls and 3rd party libraries
 
+<div style="position: relative; width: 100%; padding-bottom: 56.25%;">
+    <iframe
+        src="https://www.youtube-nocookie.com/embed/BC3c1qO_kbU"
```

Comment:
Can these be aka links? or does that break the embedded video

Reply:
We shouldn't need AKA link for YT videos; those links never change.